src/components/interaction-designer/block-editors/ContactPropertyEditor.vue

```diff
@@ -36,7 +36,7 @@
 
       <validation-message
         #input-control="{ isValid }"
-        :message-key="`block/${block.uuid}/config/set_contact_property/property_key`">
+        :message-key="`block/${block.uuid}/config/set_contact_property/0/property_key`">
```

Comment:
I think `0` will be replaced with`x` in

https://github.com/FLOIP/flow-builder/blob/feature/VMO-6421/VMO-6456/update-core-scp-block/src/store/validation/validationHelpers.ts#L94-L97

Reply:
Thanks for this @object-required , I will change